packages/apollo-server-fastify/src/ApolloServer.ts

```diff
@@ -36,17 +35,14 @@ const fileUploadMiddleware = (
   uploadsConfig: FileUploadOptions,
   server: ApolloServerBase,
 ) => (
-  req: FastifyRequest<IncomingMessage>,
-  reply: FastifyReply<ServerResponse>,
+  request: FastifyRequest<any, any, any>,
```

Comment:
Should all of these be `any` or should we be able to pass a generic or anything else here?

Reply:
Yep, I modified the code in a way that does not use any type as much as possible.